### PR TITLE
Update licenses_license_ext.sql

### LINKED ISSUE #636: ERM - Fix licenses_license_ext.sql
The derived table needs a bug fix since Metadb with folio version Lotus.

### PR DESCRIPTION
**Pull request by AlaoSUL**
Comment on Columns to provide API Documentation Descriptions

Closes #636 

```
All queries:
- [ ] Release notes added or updated in CHANGES.md
- [ ] Query runs without errors
- [ ] Query output is correct
- [ ] Query logic is clear and well documented
- [ ] Query is readable and properly indented with spaces (not tabs)
- [ ] Table and column names are in all-lowercase
- [ ] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [ ] Query begins with user documentation in comment lines
- [ ] File name is listed in `runlist.txt` after dependencies
- [ ] All columns have indexes
- [ ] Table is vacuumed and analyzed
```

### CODE REVIEW

File: sql_metadb/derived_tables/licenses_license_ext.sql [hunk 1]
```diff
@@ -1,5 +1,3 @@
--- Create a derived table that contains information about licenses from the app license
```
Comment (stdombek) [inline sql_metadb/derived_tables/licenses_license_ext.sql:1]: Please paste this line again.

**Review by stdombek — CHANGES_REQUESTED** [review]
All queries:
- [ ] Release notes added or updated in CHANGES.md
- [X] Query runs without errors
- [X] Query output is correct
- [X] Query logic is clear and well documented
- [X] Query is readable and properly indented with spaces (not tabs)
- [X] Table and column names are in all-lowercase
- [X] Quotation marks are used only where necessary

Derived tables:
- [ ] Query begins with user documentation in comment lines
- [X] File name is listed in `runlist.txt` after dependencies
- [X] All columns have indexes
- [X] Table is vacuumed and analyzed

Looks good! Please insert that one line again (see comment) and add the change to the CHANGE.md file on the root level.

**Review by stdombek — APPROVED** [review]

**Review by amzoss — APPROVED** [review]
Thanks, looks good!

All queries:
- [x] Release notes added or updated in CHANGES.md
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

File: CHANGES.md [hunk 2]
```diff
@@ -4,3 +4,4 @@
   extracts administrative notes from instance records.
 * Additional columns added to `ldp_add_columns.conf`:
   `finance_funds.fund_type_id` and `finance_transactions.description`.
+* Comment on Columns added to licenses_ext from API Documentation
```
Comment (nassibnassar) [inline CHANGES.md:7]: This should be: `licenses_license_ext`

**Pull request merged**